.github/workflows/ci.yml

```diff
@@ -23,6 +23,7 @@ jobs:
       matrix:
         os: [ ubuntu-latest, macos-latest, windows-latest ]
         arch: [ amd64 ]
+        run_supabase_tests: [ false ]
```

Comment:
Effectively, this was the case before too?

Reply:
Now adding a variant should still keep the old one.